documentation/modules/ROOT/pages/021-initial-dir-and-files.adoc

```diff
@@ -144,12 +167,6 @@ include::../examples/group_vars/dev/configure_connection_controller_credentials.
 EOF
 ----
 
-.#Content with `<CHANGE_ME>` has to be customized#
-[.console-input]
-[source,console]
-----
-export ADMIN_PRO_PASSWORD=<CHANGE_ME>
-----
```

Comment:
I'm not sure that deletion is wanted... the following cat command will use that exported variable...

Reply:
it is moved to the commands above to do it in on go for pre a pro

Reply:
As before, we could use `read -p "question: " ADMIN_${ENV}_PASSWORD` for these inputs